includes/discovery/fanspeeds/extreme.inc.php

```diff
@@ -1,6 +1,17 @@
 <?php
 
 if ($device['os'] == 'xos') {
+    // Function used to round limit values to hundreds to avoid h/w/l limits being changed on every discovery as a change of 1rpm for fan speed would cause the limit values to change since they're dynamically calculated
+    function roundme($val = 0) {
```

Comment:
Can you give this function a more descriptive name and move it to includes/common.inc.php so it can be re-used.